packages/block-editor/src/components/link-control/style.scss

```diff
@@ -166,18 +176,46 @@
 		padding: 3px 8px;
 		margin-left: auto;
 		font-size: 0.9em;
-		background-color: #f3f4f5;
+		background-color: $light-gray-200;
 		border-radius: 2px;
 	}
 }
 
+.block-editor-link-control__loading {
+	margin: $grid-size-large; // when only loading control is shown it requires it's own spacing.
+	display: flex;
+	align-items: center;
+
+	.components-spinner {
+		margin-top: 0;
+	}
+}
+
+// Separate Create button when following other suggestions.
+.components-button + .block-editor-link-control__search-create {
+	margin-top: 20px;
+	overflow: visible;
+
+	// Create fake border. We cannot use border because the button has a border
+	// radius applied to it
+	&::before {
+		content: "";
+		position: absolute;
+		top: -10px;
```

Comment:
minor: it seems to be an arbitrary value. Is it possible to define a var for this and, I guess, when it's also applied?

Reply:
👍 I've tied this to a standardised variable.